src/components/AudioPlayer/Buttons/VolumeControl.tsx

```diff
@@ -0,0 +1,75 @@
+import { useContext, useEffect, useState } from 'react';
+
+import { useSelector } from '@xstate/react';
+import useTranslation from 'next-translate/useTranslation';
+import Slider from 'react-rangeslider';
+
+import 'react-rangeslider/lib/index.css';
```

Comment:
As mentioned above, we already have `Slider` component inside `src/components/dls/Slider/index.tsx`